### PR TITLE
CO-2959 Account move fiscal year migration v11

### PR DESCRIPTION
**Pull request by quentingigon**
Not yet tested, as I had no data to do it

### CODE REVIEW

**Comment by ecino** [comment]
Migration of sbc_compassion is done, you can finish this.

**Comment by quentingigon** [comment]
The option is at the right place in the settings. I did not test the functionality because of a lack of data

File: account_move_fiscalyear/__manifest__.py [hunk 1]
```diff
@@ -10,14 +10,17 @@
...
     'name': 'Account move open bills to next fiscal year',
-    'version': '10.0.0.0.0',
+    'version': '11.0.0.0.0',
     'license': 'AGPL-3',
     'author': 'Compassion CH',
     'website': 'http://www.compassion.ch',
     'category': 'Accounting',
-    'depends': ['account'],
+    'depends': ['account',                      # source/addons/account
+                'child_compassion',
```
Comment (ecino) [inline account_move_fiscalyear/__manifest__.py]: Remove this dependency. We should not make it depedent of this module

File: account_move_fiscalyear/views/res_config_bills_view.xml [hunk 2]
```diff
@@ -1,22 +1,25 @@
 <odoo>
     <data>
         <record id="view_account_config_settings_inherit" model="ir.ui.view">
-            <field name="name">account.settings.inherit</field>
-            <field name="model">account.config.settings</field>
-            <field name="inherit_id" ref="account.view_account_config_settings"/>
+            <field name="name">res.settings.inherit</field>
+            <field name="model">res.config.settings</field>
+            <field name="inherit_id" ref="child_compassion.compassion_settings"/>
```
Comment (ecino) [inline account_move_fiscalyear/views/res_config_bills_view.xml]: Don't make this settings inherit from the settings of child_compassion. It's an accounting settings that should be placed in the accounting settings accordingly.

**Review by ecino — CHANGES_REQUESTED** [review]

**Review by ecino — APPROVED** [review]

**Pull request merged**